webrtc.html

```diff
@@ -7698,7 +7698,9 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
     session.</p>
 
     <p>Revealing IP addresses can leak location and means of
-    connection; this can be sensitive.</p>
+    connection; this can be sensitive but necessary for establishing peer to
+    peer connections. The choice of which addresses to expose is controlled by
```

Comment:
PTAL.

Thanks,
Guowei

On Mon, Dec 28, 2015 at 3:21 PM, Peter Thatcher <notifications@github.com>
wrote:

> In webrtc.html
> <https://github.com/w3c/webrtc-pc/pull/444#discussion_r48512623>:
>
> > @@ -7698,7 +7698,9 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
> >      session.</p>
> >
> >      <p>Revealing IP addresses can leak location and means of
> > -    connection; this can be sensitive.</p>
> > +    connection; this can be sensitive but necessary for establishing peer to
> > +    peer connections. The choice of which addresses to expose is controlled by
>
> Would "is determined by" be more accurate than "is controlled by"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/w3c/webrtc-pc/pull/444/files#r48512623>.
>